### PR TITLE
Switching to IdentityDivision ADAL for Upstream Feeds

### PR DESCRIPTION
**Pull request by rpdome**
Tito made a change to consolidate all of the dependencies in one upstream feed.

See: https://msazure.visualstudio.com/One/_git/AD-MFA-phonefactor-phoneApp-android/pullrequest/2206322?path=%2FPhoneFactor%2Fapp%2Fbuild.gradle&_a=overview

### CODE REVIEW

**Comment by kreedula** [comment]
Thanks @rpdome . Are you planning to do this for all other repos too ?

**Comment by shahzaibj** [comment]
It would also be good to update the Readme.md to reflect this change

**Comment by rpdome** [comment]
@kreedula https://github.com/AzureAD/ad-accounts-for-android/pull/1185

File: README.md [hunk 1]
```diff
@@ -8,18 +8,12 @@ This repository contains a build gradle and git alias commands for building ADAL
 The android related auth projects pull artifacts from public and private package repositories.  The private artifacts are published using Azure DevOps.  You will need to generate
 and store the credentials for the Identity and Aria azure devops instances.
 
-- [Aria](https://msasg.visualstudio.com/Shared%20Data/_packaging?_a=package&feed=ARIA-SDK&package=com.microsoft.applications%3Aariaandroidjavasdk-release&version=3.0.22.0&protocolType=maven)
 - [Identity](https://identitydivision.visualstudio.com/DevEx/_packaging?_a=feed&feed=AndroidADAL)
```
Comment (titorodi) [inline README.md]: Should you call out that the upstream feeds from Identity consume Aria, Intune, and PowerLift?

**Pull request merged**